README.md

```diff
@@ -20,7 +20,7 @@ $post = PostFactory::new() // Create the factory for Post objects
 The factories can be used inside [DoctrineFixturesBundle](https://symfony.com/doc/master/bundles/DoctrineFixturesBundle/index.html)
 to load fixtures or inside your tests, [where it has even more features](#using-in-your-tests).
 
-Want to watch a screencast 🎥 about it? Check out https://symfonycasts.com/foundry
+Want to watch a screencast  about it? Check out https://symfonycasts.com/foundry
```

Comment:
This should probably be reverted while merging (my terminal has a hard crash when it tries to display emojis currently, making it impossible to edit this file)

Reply:
I'll leave it out. If it causes an issue for you it will for others as well.